ssv/spectest/tests/runner/postconsensus/invalid_msg_sig.go

```diff
@@ -24,7 +24,7 @@ func InvalidMessageSignature() tests.SpecTest {
 				),
 				Duty: testingutils.TestingAttesterDuty,
 				Messages: []*types.SignedSSVMessage{
-					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(nil, testingutils.PostConsensusAttestationMsg(ks.Shares[1], 2, testingutils.TestingDutySlot))),
+					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(ks, nil, testingutils.PostConsensusAttestationMsg(ks.Shares[1], 2, testingutils.TestingDutySlot))),
```

Comment:
commenting on this line because this is where I can...
No error is generated because we don't process the post consensus message right?

Reply:
That's correct. The message is stored in a buffer and will be validated once a quorum is reached